PHYSICS/DeleteRope.md

```diff
@@ -5,10 +5,45 @@ ns: PHYSICS
 
 ```c
 // 0x52B4829281364649 0x748D72AF
-void DELETE_ROPE(int* ropeId);
+void DELETE_ROPE(int* ropeHandle);
 ```
 
+Deletes the rope with the specified handle.
+You might want to check if the rope exists before with [DOES_ROPE_EXIST](#_0xFD5448BE3111ED96).
 
 ## Parameters
-* **ropeId**: 
+* **ropeHandle**: The handle of the rope to delete
 
+## Examples
+```lua
+-- Create a rope and store the handle
+local rope = AddRope(-2096.096436, -311.906891, 14.510918, 0.0, 0.0, 0.0, 10.0, 1, 10.0, 0.0, 1.0, false, false, false, 1.0, false, 0)
+
+-- Check if the rope exists.
+if not DoesRopeExist(rope) then 
+    -- If the rope does not exist, end the execution of the code here.
+    return 
+end
+
+-- If the rope does exist, delete the rope.
+DeleteRope(rope)
+```
+
+```cs
+using CitizenFX.Core;
+using static CitizenFX.Core.Native.API;
+
+// Create a rope and store the reference
+Rope rope = World.AddRope(RopeType.ThinMetalWire, new Vector3(0f, 0f, 0f), new Vector3(0f, 0f, 0f), 0f, 0f, false);
```

Comment:
We shouldn't use C# wrappers in the docs, or if we do we should have another example below it, see https://github.com/citizenfx/natives/blob/master/ENTITY/DoesEntityExist.md for an example on how this should be done.

I would just recommend using the native version.

Reply:
Sounds good, thanks for the input!